spec/request_helper.rb

```diff
@@ -0,0 +1,25 @@
+require_relative 'spec_helper'
```

Comment:
You shouldn't need this file or to manually include the helpers in the spec file if you update to the latest `inferno_core`. You can add `:request` to a describe block to include the official version of this.

`RSpec.describe SomeClass, :request do` should automatically include them.

Reply:
Alright, I updated inferno core to 0.5.1 in the gemspec file, removed the `request_helper.rb` file, and modified the spec test accordingly.

Reply:
But I still the requester_helper file here.